modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+Argo CD has the following permissions to manage specific cluster-scoped resources which include platform operators, optional OLM operators and user management. 
```

Comment:
This should be "The default Argo CD instance installed in the openshift-gitops namespace" instead of "Argo CD"